packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -394,8 +400,6 @@ known, it can be explicitly provided to attach via the command-line, e.g.
           );
         _logger.printStatus('Waiting for a connection from Flutter on ${device.name}...');
         vmServiceUri = vmServiceDiscovery.uris;
-        // Determine ipv6 status from the scanned logs.
-        usesIpv6 = vmServiceDiscovery.ipv6;
```

Comment:
I removed this because it seems to do nothing.

It does not appear that ipv6 gets set anywhere in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/protocol_discovery.dart. It's a `final` variable and is required parameter in the constructor. 

It's set to `ipv6` in the construction 
https://github.com/flutter/flutter/blob/ae26b36509690a9dac87cda6ff9505d552a490a4/packages/flutter_tools/lib/src/commands/attach.dart#L363-L366

And `usesIpv6` is already set to `ipv6` 
https://github.com/flutter/flutter/blob/ae26b36509690a9dac87cda6ff9505d552a490a4/packages/flutter_tools/lib/src/commands/attach.dart#L294

Reply:
`ipv6` can be made private then:
https://github.com/flutter/flutter/blob/9d820aa35a22ef0365d3814462dc11f56afc13c9/packages/flutter_tools/lib/src/protocol_discovery.dart#L23-L25

```dart
    required bool ipv6,
    required Logger logger,
  })  : _logger = logger,
        _ipv6 = ipv6 {
```
https://github.com/flutter/flutter/blob/9d820aa35a22ef0365d3814462dc11f56afc13c9/packages/flutter_tools/lib/src/protocol_discovery.dart#L59
```dart
final bool _ipv6;
```